pkgs/by-name/le/leftwm-theme/package.nix

```diff
@@ -2,16 +2,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "leftwm-theme";
-  version = "unstable-2022-12-24";
+  version = "0.1.2";
 
   src = fetchFromGitHub {
     owner = "leftwm";
-    repo = pname;
-    rev = "7f2292f91f31d14a30d49372198c0e7cbe183223";
-    sha256 = "sha256-tYT1eT7Rbs/6zZcl9eWsOA651dUGoXc7eRtVK8fn610=";
+    repo = "leftwm-theme";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-tYT1eT7Rbs/6zZcl9eWsOA651dUGoXc7eRtVK8fn610=";
```

Comment:
You didn't update the hash here when you bumped the version.

```
error: hash mismatch in fixed-output derivation '/nix/store/5q2c69aqk8jl0830588sqr07004l80ss-source.drv':
         specified: sha256-tYT1eT7Rbs/6zZcl9eWsOA651dUGoXc7eRtVK8fn610=
            got:    sha256-JCNpgdV4v0DzRmELAv+UwPAZ7O2V0KI4t8zirhhQshA=
error: 1 dependencies of derivation '/nix/store/86c92zvq6cnsw6byvr3m6fzknnaqkcbl-leftwm-theme-0.1.2-vendor.tar.gz.drv' failed to build
```

Reply:
Wow, I was not able to replicate this using `nix-build -E 'with import <nixpkgs> {}; callPackage ./package.nix {}' --check` until I specifically removed the previous hash, gave no error before. Fixing now!